apps/web/src/components/settings/RegistrationSettings.tsx

```diff
@@ -0,0 +1,41 @@
+"use client";
+
+import { Button } from "@/components/shadcn/ui/button";
+import Link from "next/link";
+import { useState } from "react";
+import { Loader2 } from "lucide-react";
+
+export default function RegistrationSettings() {
+	const [isLoading, setIsLoading] = useState(false);
```

Comment:
nit: Still don't think these need loading states

Reply:
I've had times where the loading between pages has taken a while, thus to verify their click was registered.

Reply:
Have you seen this on the deployment? Or just local.

Reply:
Both. On deployment, it's usually first load, but then it's instant. Looking into it rn